deploy/crds/shipwright.io_builds_crd.yaml

```diff
@@ -0,0 +1,271 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Are these *_crd.yaml files necessary and intentional? They seem to be redundant with the non-_crd.yaml files. Having two copies might be confusing.

Reply:
Checking the file log, it came from @qu1queee trying to straight out the naming convention. Do we still need them, @qu1queee?

Reply:
Since those files are not created on this PR, we should move this conversation to a issue. Would it work for you?

Reply:
Just a quick +1 from my side to tackle unnecessary suffixes in filenames in a separate issue.